conans/tools.py

```diff
@@ -1,49 +1,210 @@
+cross_building = tools_oss.cross_building
+get_cross_building_settings = tools_oss.get_cross_building_settings
+get_gnu_triplet = tools_oss.get_gnu_triplet
+
+
+def cpu_count(*args, **kwargs):
+    return tools_oss.cpu_count(output=_global_output, *args, **kwargs)
+
+
+# from conans.client.tools.system_pm
+class SystemPackageTool(tools_system_pm.SystemPackageTool):
+    def __init__(self, *args, **kwargs):
+        super(SystemPackageTool, self).__init__(output=_global_output, *args, **kwargs)
+
+
+class NullTool(tools_system_pm.NullTool):
+    def __init__(self, *args, **kwargs):
+        super(NullTool, self).__init__(output=_global_output, *args, **kwargs)
+
+
+class AptTool(tools_system_pm.AptTool):
+    def __init__(self, *args, **kwargs):
+        super(AptTool, self).__init__(output=_global_output, *args, **kwargs)
+
+
+class YumTool(tools_system_pm.YumTool):
+    def __init__(self, *args, **kwargs):
+        super(YumTool, self).__init__(output=_global_output, *args, **kwargs)
+
+
+class BrewTool(tools_system_pm.BrewTool):
+    def __init__(self, *args, **kwargs):
+        super(BrewTool, self).__init__(output=_global_output, *args, **kwargs)
+
+
+class PkgTool(tools_system_pm.PkgTool):
+    def __init__(self, *args, **kwargs):
+        super(PkgTool, self).__init__(output=_global_output, *args, **kwargs)
+
+
+class ChocolateyTool(tools_system_pm.ChocolateyTool):
+    def __init__(self, *args, **kwargs):
+        super(ChocolateyTool, self).__init__(output=_global_output, *args, **kwargs)
+
+
+class PkgUtilTool(tools_system_pm.PkgUtilTool):
+    def __init__(self, *args, **kwargs):
+        super(PkgUtilTool, self).__init__(output=_global_output, *args, **kwargs)
+
+
+class PacManTool(tools_system_pm.PacManTool):
+    def __init__(self, *args, **kwargs):
+        super(PacManTool, self).__init__(output=_global_output, *args, **kwargs)
+
+
+class ZypperTool(tools_system_pm.ZypperTool):
+    def __init__(self, *args, **kwargs):
+        super(ZypperTool, self).__init__(output=_global_output, *args, **kwargs)
+
+
+# from conans.client.tools.win
+msvc_build_command = tools_win.msvc_build_command
+vs_installation_path = tools_win.vs_installation_path
+vswhere = tools_win.vswhere
+vs_comntools = tools_win.vs_comntools
+find_windows_10_sdk = tools_win.find_windows_10_sdk
+vcvars_dict = tools_win.vcvars_dict
+vcvars = tools_win.vcvars
+escape_windows_cmd = tools_win.escape_windows_cmd
+get_cased_path = tools_win.get_cased_path
+MSYS2 = tools_win.MSYS2
+MSYS = tools_win.MSYS
+CYGWIN = tools_win.CYGWIN
+WSL = tools_win.WSL
+SFU = tools_win.SFU
+unix_path = tools_win.unix_path
+run_in_windows_bash = tools_win.run_in_windows_bash
+
+
+def build_sln_command(*args, **kwargs):
+    return tools_win.build_sln_command(output=_global_output, *args, **kwargs)
+
+
+def vcvars_command(*args, **kwargs):
+    return tools_win.vcvars_command(output=_global_output, *args, **kwargs)
+
+
+
+# Ready to use objects.
+try:
+    os_info = OSInfo()
```

Comment:
This seems not used along the PR. The access to ``os_info`` seems to have been replaced by creation of ``OSInfo()`` per user.

Reply:
...but an existing recipe can be using it (https://docs.conan.io/en/latest/reference/conanfile/methods.html#system-requirements)